src/bsp/platform/cc3200/xi_bsp_io_net_cc3200.c

```diff
@@ -269,24 +264,27 @@ xi_bsp_io_net_state_t xi_bsp_io_net_select( xi_bsp_socket_events_t* socket_event
 
         if ( 1 == socket_events->in_socket_want_read )
         {
-            FD_SET( socket_events->socket, &rfds );
-            max_fd_read =
-                socket_events->socket > max_fd_read ? socket_events->socket : max_fd_read;
+            FD_SET( socket_events->xi_socket, &rfds );
```

Comment:
I use the SublimeText and the formatter definitions are same as in file .clang-format